model/annotations/task_annotations.go

```diff
@@ -211,15 +211,23 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 func PatchAnnotation(a *TaskAnnotation, userDisplayName string, upsert bool) error {
 	existingAnnotation, err := FindOneByTaskIdAndExecution(a.TaskId, a.TaskExecution)
 	if err != nil {
-		return errors.Wrapf(err, "finding task annotation for task id '%s'", a.TaskId)
+		return errors.Wrapf(err, "finding task annotation")
```

Comment:
are we sure we want to remove the task id?

Reply:
it seems that that's what kim did in her fixes in other functions in this file so i wanted to stay consistent

Reply:
her fix was more about voiding phrases like "failed to" to make things less noisy. You can read more here: https://jira.mongodb.org/browse/EVG-16501. 
I don't think we should be removing the task id specifically.

Reply:
added taskid and execution